notebooks/lesson10_clusters.ipynb

```diff
@@ -0,0 +1,842 @@
+{
```

Comment:
<ul><li>capitalise the docstrings?</li><li>drop the "YOUR CODE" piece or make it explicitly an exercise? sorry just saw the exercise below - maybe it is simpler to define the manhattan distance mathematically as an equation and get them to implement that? the current flow is a bit confusing ...</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>The idea was that i show it to them during the lecture and define it mathematically and then they do it themselves. I added the mathematical definition.</p>